### ISSUE TITLE
Error while loading C#(Blazor WASM) projects, csdevkit extension

### ISSUE THREAD

**Issue by SadToBeNep** [dotnet/vscode-dotnet-runtime#1063]
The issue text was copied to the clipboard.  Please paste it into this window.

**Repro steps:**

1.

**Extension Version:** 1.0.0
**Error Message:** The "file" argument must be of type string. Received an instance of Object
**Error Stack:** TypeError: The "file" argument must be of type string. Received an instance of Object
        at new NodeError (node:internal/errors:387:5)
        at validateString (node:internal/validators:121:11)
        at normalizeSpawnArguments (node:child_process:527:3)
        at Object.spawnSync (node:child_process:825:8)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1095661
        at Generator.next (<anonymous>)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1092018
        at new Promise (<anonymous>)
        at n (/home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1091763)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1095584
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:37962
        at Generator.next (<anonymous>)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:36621
        at new Promise (<anonymous>)
        at n (/home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:36366)
        at t.callWithErrorHandling (/home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:37910)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1095578
        at Generator.next (<anonymous>)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1092018
        at new Promise (<anonymous>)
        at n (/home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1091763)
        at /home/anonymous/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-1.6.0/dist/extension.js:2:1095512
        at e.$vL.h (/opt/visual-studio-code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:64085)
        at e.$vL.g (/opt/visual-studio-code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:63073)
        at e.$vL.executeCommand (/opt/visual-studio-code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:62973)
        at Object.executeCommand (/opt/visual-studio-code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:117:20971)
        at t.DotnetRuntimeExtensionResolver.<anonymous> (/home/anonymous/.vscode/extensions/ms-dotnettools.csharp-2.0.249-linux-x64/dist/extension.js:2:1010140)
        at Generator.next (<anonymous>)
        at s (/home/anonymous/.vscode/extensions/ms-dotnettools.csharp-2.0.249-linux-x64/dist/extension.js:2:1008463)

I got this error msg while trying to load a C# project under Linux

Specifications that could help:
Arch Linux x64

uname -a 
```
Linux lily 6.4.3-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 11 Jul 2023 05:13:39 +0000 x86_64 GNU/Linux
```

dotnet --list-sdks
```
7.0.306 [/usr/share/dotnet/sdk]
8.0.100-preview.5.23303.2 [/usr/share/dotnet/sdk]
```

code --version
```
1.80.1
74f6148eb9ea00507ec113ec51c489d6ffb4b771
x64
```
/tmp/csdevkit/b083ca93.json
```
{"common.vscodesessionid":"b083ca93-a459-4e2a-8e49-d9db70f818d21689475208415","common.extname":"ms-dotnettools.csdevkit","common.extversion":"0.3.2","common.os":"linux","common.platformversion":"6.4.3","common.vscodemachineid":"aaaefeb2cb22a005961c5382bd01af6275b9f5765610de3a4b259202a27e8906","common.vscodeversion":"1.80.1","common.product":"desktop","GlobalStorageUri":"/home/USR/.config/Code/User/globalStorage/ms-dotnettools.csdevkit"}
```

It's my first issue, so not sure what else is needed, I collected all informations that I found valuabe

**Comment by baronfel** [comment]
Thanks for the detailed report @SadToBeNep - I'm closing this as a duplicate of #1007, but there's an issue against the C# extension linked from that issue, and the C# extension team has been contacted internally to dig into what's going on. In the meantime, version 2.0.248 of the C# extension doesn't have this error so you can use it to unblock yourself in the meantime.
